bot.py

```diff
@@ -2,18 +2,39 @@
 import asyncio
 import configparser
 import json
+import os
+import os.path
 import random
 import time
 from datetime import datetime
+from pathlib import Path
 
 import requests
 from requests_html import HTMLSession
 from twitchio.ext import commands
 
+# Criar a pasta caso não exista.
+if not os.path.exists('files'):
+    os.mkdir('files')
+    print('Pasta criada.')
+
+os.chdir('files')  # Muda para a pasta files.
+Path("commands.json").touch()  # Cria o arquivo caso não tenha ainda.
+
+# Acessar o arquivo e recuperar os comandos anteriormente criados.
+with open("commands.json", 'r', encoding='UTF-8') as file:
+    try:
+        namecommand = json.load(file)
+    except ValueError:
```

Comment:
`json#load` não lança `ValueError`.

De acordo com as [docs](https://docs.python.org/3/library/json.html#json.load) pode ser lançada uma [`JSONDecodeError`](https://docs.python.org/3/library/json.html#json.JSONDecodeError) se o conteúdo do arquivo não for um JSON válido.

Reply:
eu tentei colocar o JSONDecodeError, porém levantou outro erro, dai eu testei com o ValueError e funcionou certinho, dai deixei ele